test_static.cpp

```diff
@@ -35,10 +35,12 @@
 #define ISPC_IS_WASM
 #elif defined(_WIN32) || defined(_WIN64)
 #define ISPC_IS_WINDOWS
-#elif defined(__linux__)
+#elif defined(__linux__) || defined(__FreeBSD__)
```

Comment:
Please add a check of FreeBSD platform in test_static_l0.cpp and report that platform is not supported.

Reply:
fixed, please have a look if it's what you expected